pyccel/codegen/f2py.py

```diff
@@ -42,7 +42,7 @@ def compile_f2py( filename, *,
     if compiler == 'gfortran' or compiler == 'gcc':
         _vendor = 'gnu95'
 
-    if compiler == 'gcc':
```

Comment:
Isn't this condition covered by line 42 above?

Reply:
Oops, I must have forgotten to delete the second condition. I haven't managed to get f2py to work with gcc anyway though :/